common/rest/request_test.go

```diff
@@ -1,13 +1,16 @@
-package rest
+package rest_test
```

Comment:
**Callout**: Changed to `rest_test` to prevent import cycle error when importing `models` package.